### PR TITLE
update bazel.repos according to latest branches

### PR DESCRIPTION
**Pull request by sschaal2**
bazel branches have advanced ....

### CODE REVIEW

File: example/bazel.repos [hunk 1]
```diff
@@ -6,64 +6,64 @@ repositories:
...
   ign_transport:
     type: git
     url: https://github.com/ignitionrobotics/ign-transport
-    version: bazel
+    version: bazel-transport9
   dart:
     type: git
     url: https://github.com/azeey/dart
-    version: bazel
+    version: bazel-azeey
```
Comment (mahiuchun) [inline example/bazel.repos:61]: use `master` as it is merged recently.

File: example/bazel.repos [hunk 2]
```diff
@@ -6,64 +6,64 @@ repositories:
...
   ogre2:
     type: git
     url: https://github.com/osrf/ogre-2.1-release
     version: bazel
```
Comment (mahiuchun) [inline example/bazel.repos:65]: same here.

File: example/bazel.repos (hunk 1 above)
Comment (mjcarroll) [reply example/bazel.repos:61]: Actually, can also be updated to use: https://github.com/ignition-forks/dart

File: example/bazel.repos (hunk 2 above)
Comment (mjcarroll) [reply example/bazel.repos:65]: And here: https://github.com/ignition-forks/ogre-2.1-release

**Pull request merged**